taca/storage/storage.py

```diff
@@ -24,19 +24,34 @@ def cleanup_nas(seconds):
 
     :param int seconds: Days/hours converted as second to consider a run to be old
     """
+    couch_info = CONFIG.get('statusdb')
```

Comment:
maybe worth checking if these fields are in the config in order to control the termination